rpcs3/Emu/RSX/Common/texture_cache.h

```diff
@@ -2468,11 +2471,43 @@ namespace rsx
 			return m_predictor;
 		}
 
+
+		/**
```

Comment:
This moving around of code is not necessary. I get the idea to group similar methods, but preserve git history as much as possible.